inference-engine/src/vpu/graph_transformer/src/parsed_config.cpp

```diff
@@ -216,6 +236,9 @@ IE_SUPPRESS_DEPRECATED_END
     if (const auto envVar = std::getenv("IE_VPU_NUMBER_OF_SHAVES_AND_CMX_SLICES")) {
         _compileConfig.numSHAVEs = _compileConfig.numCMXSlices = std::stoi(envVar);
     }
+    if (const auto envVar = std::getenv("IE_VPU_TILING_CMX_LIMIT_KB")) {
+        _compileConfig.tilingCMXLimitKB = std::stoi(envVar);
```

Comment:
Should you use `preprocessCompileOption` here?

Reply:
@gladilov-gleb We also don't use `preprocess` here for other compile options. Looks like `#ifndef NDEBUG` matter.

Reply:
Do you think we should use it there? I don't think `NDEBUG` matters here.

Reply:
>  I don't think NDEBUG matters here.  
I suppose, it was made for debugging.

> Do you think we should use it there?  
Do you suggest changing this piece of code with compile options?

Reply:
I suggest to replace `std::stoi` with preprocess even for options for debug only

Reply:
I hope you agree not to change it:
`_compileConfig.dumpAllPasses = std::stoi(envVar) != 0;`